master/internal/config/provconfig/config.go

```diff
@@ -151,3 +151,18 @@ func (c *Config) InitMasterAddress() error {
 	c.MasterURL = (&url.URL{Scheme: scheme, Host: fmt.Sprintf("%s:%s", host, port)}).String()
 	return nil
 }
+
+// Printable returns a printable object.
+func (c Config) Printable() Config {
+	res := c
```

Comment:
Is this necessary, given that the receiver here is a value rather than a pointer?